Resources/doc/entity-listeners.rst

```diff
@@ -46,7 +45,7 @@ Full example:
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
             <services>
-                <service id="user_listener" class="UserListener">
+                <service id="UserListener">
```

Comment:
This will be confusing for unaware users, because this doesn't look like FCQN anymore. We should change it to eg. `App\UserListener`